docs/reference/text-structure/apis/find-structure.asciidoc

```diff
@@ -99,6 +99,12 @@ specified, the name of the timestamp field in the Grok pattern must match
 "timestamp". If `grok_pattern` is not specified, the structure finder creates a
 Grok pattern.
 
+`ecs_compatibility`::
+(Optional, string) The mode of compatibity with ECS compliant Grok patterns.
+When the structure finder creates a Grok pattern you can use this parameter to
+specify whether to use ECS Grok patterns instead of the legacy ones. Valid values
+are `disabled` and `v1`. The default value is `disabled`.
+
```

Comment:
```suggestion
`ecs_compatibility`::
(Optional, string) The mode of compatibility with ECS compliant Grok patterns.
Use this parameter to specify whether to use ECS Grok patterns instead of
legacy ones when the structure finder creates a Grok pattern. Valid values
are `disabled` and `v1`. The default value is `disabled`. This setting primarily
has an impact when a whole message Grok pattern such as `%{CATALINALOG}`
matches the input. If the structure finder identifies a common structure but
has no idea of meaning then generic field names such as `path`, `ipaddress`,
`field1` and `field2` are used in the `grok_pattern` output, with the intention
that a user who knows the meanings rename these fields before using it.
```